src/services/drive/database/driveLocalDB.ts

```diff
@@ -134,7 +133,7 @@ class DriveLocalDB {
     return sqliteService.executeSql(DRIVE_DB_NAME, folderRecordTable.statements.deleteById, [folderId]);
   }
 
-  public async deleteItem({ id, isFolder }: { id: number; isFolder: boolean }) {
+  public async deleteItem({ id }: { id: number; isFolder: boolean }) {
```

Comment:
Maybe remove isFolder from the type as well 🤔

Reply:
Done